manager/app/src/main/java/me/weishu/kernelsu/ui/screen/Home.kt

```diff
@@ -200,6 +200,26 @@ private fun InfoCard() {
             val contents = StringBuilder()
             val uname = Os.uname()
 
+            val GetSELinuxStatus = Runtime.getRuntime().exec("getenforce")
+            GetSELinuxStatus.waitFor()
+            val GetSELinuxStatusOut = GetSELinuxStatus.errorStream.bufferedReader().readLine() ?: GetSELinuxStatus.inputStream.bufferedReader().readLine()
```

Comment:
stream not closed?

Reply:
thank you for correction i will close it after getting the results